mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,19 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a,b,c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+  name = "Sonjide";
+  age = 27;
+  return "Hello, my name is " + name + " and I am " + age + " years old";
+}
 
 function getTotal(a, b) {
-  total = a ++ b;
-
-  return "The total is total";
+  a = 23;
+  b = 5;
+  return "The total is " + (a + b);
 }
```

Comment:
This wants to be something like the following:

```
function getTotal(a, b) {
  const total = a + b;
  return "The total is total " + total;
}
```

By reassigning a and b in the function means you are throwing away the values being passed in